include/device.h

```diff
@@ -692,11 +692,27 @@ int device_busy_check(const struct device *chk_dev);
 #define Z_DEVICE_EXTRA_HANDLES(...)				\
 	FOR_EACH_NONEMPTY_TERM(IDENTITY, (,), __VA_ARGS__)
 
+/* If a device has power management, this macro defines a pointer to a device
```

Comment:
Nitpick: This is comment seems to imply that only devices supporting power management get to define a pointer.

```suggestion
/* If device power management is enabled, this macro defines a pointer to a device
```

Reply:
thanks, fixed